test/categories.test.js

```diff
@@ -0,0 +1,209 @@
+const chai                                  = require('chai');
+const chaiHttp                              = require('chai-http');
+const faker                                 = require('faker');
+const app                                   = require('../server');
+const createUserAdminAndLogin               = require('./utils/createUserAdminAndLogin');
+const {createCategory, deleteAllCategories} = require('./utils/createCategory');
+
+chai.use(chaiHttp);
+chai.should();
+
+const expect = chai.expect;
+
+let credentials;
+
+describe('Category', () => {
+    beforeEach(async () => {
+        await deleteAllCategories();
+        credentials = await createUserAdminAndLogin();
+    });
+
+    describe('POST /api/v2/categories', () => {
+        it('Create category and get it with the id', async () => {
+            const category = await createCategory();
+            const res      = await chai.request(app)
+                .post('/api/v2/categories')
+                .set('authorization', credentials.token)
+                .send({PostBody: {filter: {_id: category._id}, limit: 99}});
+            expect(res).to.have.status(200);
+            expect(res.body.datas[0].code).be.equals(category.code);
+        });
+        it('Create category and get it with the id - w/o authentication', async () => {
+            const category = await createCategory();
+            const res      = await chai.request(app)
+                .post('/api/v2/categories')
+                .send({PostBody: {filter: {_id: category._id}, limit: 99}});
+            expect(res).to.have.status(200);
+            expect(res.body.datas[0].code).be.equals(category.code);
+        });
+        it('Create category and get it with the id - w/o the good id', async () => {
+            await createCategory();
+            const res = await chai.request(app)
+                .post('/api/v2/categories')
+                .set('authorization', credentials.token)
+                .send({PostBody: {filter: {_id: '111111111111111111111111'}, limit: 99}});
+            expect(res).to.have.status(200);
```

Comment:
weird behavior

Reply:
Yes. Big change for that, don't change it please